src/main/java/com/bootcamp/demo/model/User.java

```diff
@@ -76,5 +81,16 @@ public void setPassword(String password) {
         this.password = password;
     }
 
-
+    @Override
+    public String toString() {
+        return "User{" +
+                "firstName='" + firstName + '\'' +
+                ", lastName='" + lastName + '\'' +
+                ", email='" + email + '\'' +
+                ", password='" + password + '\'' +
+                ", phoneNumber='" + phoneNumber + '\'' +
+                ", gender=" + gender +
+                ", id=" + id +
+                '}';
+    }
```

Comment:
As an advice, when you want to apply an easier method to log the model information in the toString method, you could do the following:
* add the dependency to the library org.apache.commons.lang in the pom.xml file
* whenever needed, import the following:

```
import static org.apache.commons.lang.builder.ToStringBuilder.reflectionToString;
import static org.apache.commons.lang.builder.ToStringStyle.SHORT_PREFIX_STYLE;
```
and in the class model, the toString operation would become:
```
    @Override
    public String toString() {
        return reflectionToString(this, SHORT_PREFIX_STYLE);
    }
```
In that manner, the library will handle all the attributes, and their format, and no manual updates would be needed if at a later point, an attribute is added or removed to the model.

Reply:
Thank you for the advice. I never used it . I'm looking forward to test it tomorrow.

Reply:
I integrated it in the AbstractModel. 
commit d22d632c95f32e0a79d97d742265c0a2407231c1